port/unix/omrsl.c

```diff
@@ -45,11 +45,14 @@
 #include <errno.h>
 #include <dlfcn.h>
 #include "ut_omrport.h"
+#include "omrport.h"
 
 /* Length of string buffers. */
 #define MAX_STRING_LENGTH 1024
 #define MAX_ERR_BUF_LENGTH 512
 
+/* Max Length of Library name */
+#define READ_CHUNK_SIZE 4096
```

Comment:
The comment isn't really accurate, should end with a period, and might be better placed just before the function that uses it.